preprocessors/charts.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/charts.schema.json",
+    "type": "object",
+    "title": "Charts preprocessor",
+    "description": "Extracts additional information from the datapoints present in the graph",
```

Comment:
This only has the possibility of encoding information on a graph with two quantitative axes...please update the description/title/ID to make this name more accurate to what is encoded or, if this is meant to be a generic data type that will be extended to meet a larger variety of graphs, include a mention that this is not complete.